src/sinks/redis/service.rs

```diff
@@ -0,0 +1,103 @@
+use std::task::{Context, Poll};
+
+use redis::aio::ConnectionManager;
+
+use crate::sinks::{prelude::*, util::sink::Response};
+
+use super::{config::Method, RedisRequest, RedisSinkError};
+
+#[derive(Clone)]
+pub struct RedisService {
+    pub(super) conn: ConnectionManager,
+    pub(super) data_type: super::DataType,
+}
+
+impl Response for Vec<bool> {
+    fn is_successful(&self) -> bool {
+        self.iter().all(|x| *x)
+    }
+}
```

Comment:
I don't believe this is used anymore because we changed the service's response type from `Vec<bool>`.
```suggestion
```